### PR TITLE
feat: update eKuiper plugin doc

### PR DESCRIPTION
**Pull request by eeff**

### CODE REVIEW

**Review by fengzeroz — APPROVED** [review]

File: en_US/north-apps/ekuiper/overview.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+This section describes low level communication details between Neuron and
+eKuiper. You may skip this section without problem.
+
+#### Data upload
+
+The Neuron eKuiper plugin publishes data collected from devices as JSON once
+connected to the eKuiper.
+Data published to eKuiper have the following fields:
+* "timestamp": unix timestamp when the data is collected
```
Comment (lixiumei123) [inline en_US/north-apps/ekuiper/overview.md:56]: remove double quotation marks.

File: en_US/north-apps/ekuiper/overview.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+#### Device control
+
+eKuiper may control devices via Neuron by sending *write commands* using the
+Neuron sink, and Neuron will write data to the devices upon receiving the
+commands.
+
+The *write command* should be JSON data with the following fields:
+* "node_name": the name of a southbound node.
```
Comment (lixiumei123) [inline en_US/north-apps/ekuiper/overview.md:88]: remove double quotation marks.

File: zh_CN/north-apps/ekuiper/faq.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## 如何写 eKuiper 规则以满足我的需求
+
+请参考 [eKuiper Doc](https://ekuiper.org/docs/en/latest)
+
+## *data-stream-processing* 北向节点处于未连接状态，而 eKuiper 运行正常
+
+确保您创建了使用 eKuiper Neuron 源的规则，并且 eKuiper 会延迟连接直到规则被启动。
+
+## 我怎么知道 eKuiper 成功从 Neuorn 采集到了数据
```
Comment (lixiumei123) [inline zh_CN/north-apps/ekuiper/faq.md:11]: 如何查看 eKuiper 是否成功从 Neuron 采集到数据

File: zh_CN/north-apps/ekuiper/faq.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# 常见问题
+
+## 如何写 eKuiper 规则以满足我的需求
+
+请参考 [eKuiper Doc](https://ekuiper.org/docs/en/latest)
```
Comment (lixiumei123) [inline zh_CN/north-apps/ekuiper/faq.md:5]: eKuiper Docs

File: zh_CN/north-apps/ekuiper/overview.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+### 内部实现
+
+本节描述了 Neuron 和 eKuiper 之间的底层通信细节。如果您不关心这些细节，可以跳过本节。
+
+#### 数据上报
+
+Neuron eKuiper 插件连接到 eKuiper 后，将从设备收集的数据以JSON形式发布。
+发布到 eKuiper 的数据具有以下字段：
+* "timestamp": 数据采集时的UNIX时间戳。
```
Comment (lixiumei123) [inline zh_CN/north-apps/ekuiper/overview.md:41]: remove double quotation marks.

File: zh_CN/north-apps/ekuiper/overview.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  }
+}
+```
+
+#### 反向控制
+
+eKuiper 可以通过使用 Neuron 动作发送写命令来控制设备，Neuron 在接收到命令后将向设备写入数据。
+
+写命令应该是一个带有以下字段的 JSON 数据：
+* "node_name": 某个南向节点名字。
```
Comment (lixiumei123) [inline zh_CN/north-apps/ekuiper/overview.md:70]: remove double quotation marks.

File: zh_CN/north-apps/ekuiper/faq.md (hunk 3 above)
Comment (eeff) [reply zh_CN/north-apps/ekuiper/faq.md:11]: Fix

**Review by lixiumei123 — APPROVED** [review]

**Pull request merged**